app/src/main/java/com/example/musictoned/editRoutine/EditRoutineScreen.kt

```diff
@@ -177,7 +178,10 @@ private fun TopBar(
                     containerColor = Color(0xFF5E60CE),
                     contentColor = Color(0xFFFFFFFF),
                 ),
-                onClick = { onNavigateToRoutine(0) }
+                onClick = {
+                    onNavigateToRoutine(0)
```

Comment:
So once we're done editing shouldn't we route to the routine we're editing instead of defaulting to routing to routine 0?

Reply:
And do you think we should save first before we route?

Reply:
Good catch, we should be navigating with the current object id

Reply:
I'll merge this for now, we can always circle back later